resources/icons/16px/arrows-arc.svg

```diff
@@ -0,0 +1,10 @@
+<svg viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
+<g clip-path="url(#clip0_5_337)">
```

Comment:
Can we re-export these icons without the `clip-path`?
```svg
<svg viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
<path d="M3.39642 0.146447C3.59168 -0.0488155 3.90826 -0.0488155 4.10353 0.146447L7.28551 3.32843C7.48077 3.52369 7.48077 3.84027 7.28551 4.03553C7.09025 4.2308 6.77366 4.2308 6.5784 4.03553L4.24637 1.70351L4.24636 1.70623C4.23808 3.5013 4.23183 4.85464 4.43039 6.02088C4.66309 7.38759 5.17704 8.47621 6.3526 9.6455C7.53497 10.8216 8.56871 11.3321 9.89872 11.565C11.0476 11.7661 12.4058 11.7607 14.2884 11.7531C14.291 11.7531 14.2935 11.7531 14.296 11.7531L11.9645 9.42157C11.7692 9.22631 11.7692 8.90973 11.9645 8.71447C12.1597 8.5192 12.4763 8.5192 12.6716 8.71447L15.8536 11.8964C16.0488 12.0917 16.0488 12.4083 15.8536 12.6036L12.6716 15.7855C12.4763 15.9808 12.1597 15.9808 11.9645 15.7855C11.7692 15.5903 11.7692 15.2737 11.9645 15.0784L14.2894 12.7535C12.4229 12.7616 10.9639 12.7667 9.72627 12.55C8.18128 12.2795 6.96501 11.6651 5.64739 10.3545C4.32296 9.03714 3.7119 7.75882 3.44458 6.18872C3.23171 4.93842 3.2372 3.48937 3.24599 1.71109L0.921546 4.03553C0.726284 4.2308 0.409702 4.2308 0.21444 4.03553C0.0191777 3.84027 0.0191777 3.52369 0.21444 3.32843L3.39642 0.146447Z"/>
</svg>
```
<img src="https://github.com/user-attachments/assets/058ab3d9-a590-4383-befc-81f216ed9aae" alt="arrows-arc" width="256" />